ikarus/finiteelements/ferequirements.hh

```diff
@@ -59,7 +59,7 @@ namespace Ikarus {
             geometricstiffness,
             stiffnessdiffBucklingVector,
             microMagneticHessian,
-            mass
+            linearMass
```

Comment:
What do you mean by linear?

Reply:
That was a suggestion, I think that is in constrast to a mass matrix that is evaluated at the deformed configuration